lib/pages/save_video/widgets/save_button.dart

```diff
@@ -203,4 +127,150 @@ class _SaveButtonState extends State<SaveButton> {
       ),
     );
   }
+
+  Future<void> _editWithFFmpeg(bool isGeotaggingEnabled, BuildContext context) async {
+    // Positions to render texts for the (x, y co-ordinates)
+    // According to the ffmpeg docs, the x, y positions are relative to the top-left side of the output frame.
+    final String datePosY = widget.isTextDate ? 'h-th-40' : '40';
+    final String datePosX = widget.isTextDate ? '40' : 'w-tw-40';
+    const String locPosY = 'h-th-40';
+    const String locPosX = 'w-tw-40';
+
+    final double dateTextSize = MediaQuery.of(context).size.width * 0.032 * 2.5;
```

Comment:
I think we should set a default size for this, since video resolution is the same for all devices. This conditional size would result too large text in some devices (i.e tablets).